Classes/shared/SQKCoreDataOperation/SQKCoreDataOperation.m

```diff
@@ -56,21 +63,29 @@ - (void)start
         [self didChangeValueForKey:@"isFinished"];
         return;
     }
-
+    
     [self willChangeValueForKey:@"isExecuting"];
     self.sqk_executing = YES;
     [self didChangeValueForKey:@"isExecuting"];
-
+    
     self.managedObjectContextToMerge = [self.contextManager newPrivateContext];
+    
+    // We are going to manage merging the context ourselves
     self.managedObjectContextToMerge.shouldMergeOnSave = NO;
+    
     [self.managedObjectContextToMerge performBlockAndWait:^{
         [self performWorkWithPrivateContext:self.managedObjectContextToMerge];
     }];
 }
 
 - (BOOL)isConcurrent
```

Comment:
I'm not sure if you should have both `isConcurrent` and `isAsynchronous`, but as long as they agree it should be okay.